draft-ietf-quic-transport.md

```diff
@@ -3577,6 +3588,10 @@ connection IDs gives clients some assurance that the server received the packet
 and that the Version Negotiation packet was not generated by an off-path
 attacker.
 
+As future versions of QUIC may support Connection IDs larger than the version 1
+limit, Version Negotiation Packets will sometimes exceed the Connection ID
```

Comment:
```suggestion
limit, Version Negotiation Packets MAY carry Connection IDs that are longer than 48 bytes.
```

Reply:
s/MAY/could - it's not the place of this specification to give permission to break its own rules.